assembl/models/vote_session.py

```diff
@@ -15,7 +16,7 @@
 
 
 class VoteSession(
-    LangstringsBase(langstrings_names),
+    LangstringsBase(langstrings_names, "VoteSession"),
```

Comment:
Better approach, `models.VoteSession.__name__`, keeps things DRY

Reply:
Can't access VoteSession name in it's definition without doing dark magic :(
```
class foo(object):
    _name = traceback.extract_stack()[-1][2]
```

Reply:
Woah, that's surprising.

None of these work? 
https://stackoverflow.com/questions/510972/getting-the-class-name-of-an-instance-in-python